panels/templates/other_panelists_form.html

```diff
@@ -52,7 +52,7 @@
         <h2>Other Panelist #{{ loop.index }}</h2>
         {{ panel_macros.panelist_form(
             op,
-            suffix='_' + loop.index1|string,
```

Comment:
This is the change that I believe fixes #119. If you submit additional "Other Panelists" with bad data, when the form was re-rendered after the POST, all of the "Other Panelists" form fields would have the wrong name. They'd all be like `first_name_` without the integer suffix. That will cause every subsequent submission of the form to fail!

Reply:
Oh no! What a tricky bug, nice find!